test/mod.test.ts

```diff
@@ -148,10 +148,8 @@ Deno.test({
 Deno.test({
   name: "[channel] create a channel in a guild",
   async fn() {
-    const guild = cache.guilds.get(tempData.guildID);
-    if (!guild) throw new Error("Guild not found");
```

Comment:
why removing that? there we can check indirectly if guild cache is working.

Reply:
The test case is not to check whether or not, cache is working.

Reply:
then we should add that in another pr